pkg/signup/service/signup_service_test.go

```diff
@@ -1093,7 +1093,8 @@ func (s *TestSignupServiceSuite) TestGetSignupDeactivated() {
 	// given
 	s.ServiceConfiguration(configuration.Namespace(), true, "", 5)
 
-	us := s.newUserSignupCompleteWithReason("Deactivated")
+	us := s.newUserSignupComplete()
+	us.Status.Conditions = deactivated()
```

Comment:
Maybe I'm missing something but this test doesn't seem to check the `signup` payload status in lines 1132-1133. So it doesn't really test the changed signup service. Could you please add it to the test? Check the status.